tests/compile-fail/panic1.rs

```diff
@@ -1,5 +1,5 @@
-//error-pattern: the evaluated program panicked
-
+// error-pattern: the evaluated program panicked
+// compile-flags: -C panic=abort
```

Comment:
What happens in all of these cases with the default flags? Do we have any test for a "normal" panic?

Reply:
I added a test for panicking with the default flags.